database/gdb/gdb_core_ctx.go

```diff
@@ -26,6 +26,13 @@ type internalCtxData struct {
 
 const (
 	internalCtxDataKeyInCtx gctx.StrKey = "InternalCtxData"
+
+	// IgnoreResultInCtx
+	// This option is only available in ClickHouse.
+	// Because ClickHouse does not support fetching insert/update results and returns errors when executed
+	// So need to ignore the results to avoid triggering errors
+	// Rather than ignoring errors after they are triggered
+	IgnoreResultInCtx gctx.StrKey = "IgnoreResult"
```

Comment:
1. `ignoreResultKeyInCtx`
2. 注释改为：
```
// `ignoreResultKeyInCtx` is a mark for some db drivers that do not support `RowsAffected` function, 
// for example: `clickhouse`. The `clickhouse` does not support fetching insert/update results, 
// but returns errors when execute `RowsAffected`. It here ignores the calling of `RowsAffected`
// to avoid triggering errors, rather than ignoring errors after they are triggered.
```